PowerPointLabs/PowerPointLabs/PasteLab/PasteLabMain.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Windows;
+
+using PowerPointLabs.Utils;
+
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace PowerPointLabs.PasteLab
+{
+    public class PasteLabMain
+    {
+        public static void PasteToFillSlide(Models.PowerPointSlide slide, float width, float height)
+        {
+            if (IsClipboardEmpty())
+            {
+                return;
+            }
+
+            PowerPoint.ShapeRange pastedObject = slide.Shapes.Paste();
+            for (int i = 1; i <= pastedObject.Count; i++)
+            {
+                var shape = new PPShape(pastedObject[i]);
+                shape.AbsoluteHeight = height;
+                shape.AbsoluteWidth = width;
+                shape.VisualTop = 0;
+                shape.VisualLeft = 0;
+            }
+        }
+
+        internal static bool IsClipboardEmpty()
+        {
+            return Clipboard.GetDataObject() == null;
```

Comment:
I see where you are coming from. Currently this method is used in the `PasteToFillSlide` method and in the Enabled handler.

There are several ways to approach this:
1. `PasteToFillSlide` takes in the status of the Clipboard (status may not reflect the actual status of the clipboard, as pasting is done here)
2. `PasteToFillSlide` assumes that the Clipboard has been checked (riskier, as we need to guard the callers)